cert_manager/certbot.py

```diff
@@ -0,0 +1,100 @@
+"""The main CertificateManager class that interacts with Certbot."""
```

Comment:
Nit: bad copy pasta :p

Reply:
Not even that – I renamed both the class and the file and moved some code out, but did not update the docstring.